<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QWidget" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>6</number>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="headLabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>头像</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>标题</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="previewLabel">
           <property name="text">
            <string>预览.....</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="msgBrowser">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QToolButton" name="sendFileBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="toolTip">
            <string>发送文件</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/source/ico/sendfile.ico</normaloff>:/source/ico/sendfile.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>248</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sizeComboBox">
           <property name="minimumSize">
            <size>
             <width>41</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>41</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="currentText">
            <string>9</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="boldToolBtn">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>加粗</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/source/ico/bold.ico</normaloff>:/source/ico/bold.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="italicToolBtn">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>倾斜</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/source/ico/italic.ico</normaloff>:/source/ico/italic.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="underlineToolBtn">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>下划线</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/source/ico/underline.ico</normaloff>:/source/ico/underline.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="colorToolBtn">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>字体颜色</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/source/ico/color.ico</normaloff>:/source/ico/color.ico</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="msgTextEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="autoFormatting">
          <set>QTextEdit::AutoNone</set>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendMsgButton">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="sideStackedWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QWidget" name="filePage">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>290</width>
              <height>395</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ChatForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
